lib/chef/provider/package/windows/exe.rb

```diff
@@ -0,0 +1,125 @@
+#
```

Comment:
going to make a general style comment on this file:
you should avoid accessing variables throught `@`. The reason is because if you mistype something, ruby will happily continue along.